QuizZero.Maven/src/main/java/QuizZero.java

```diff
@@ -0,0 +1,96 @@
+import static java.lang.System.exit;
+
+public class QuizZero {
+
+    public static void main(String[] args) {
+        QuizZero qz = new QuizZero();
+
+        int score = qz.test();
+        exit(score);
+    }
+
+    public int test() {
+        int runningTotal;
+        return 15 - 15;
+    }
+
+
+    // return zero
+    public static  int Qzero(int a, int b) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.